### PR TITLE
feat(GM-742): Build YARA-X in separate pipeline

### PR DESCRIPTION
**Pull request by jknyspel-catalogicsoftware**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+// TODO: How to do this???
+if(! env.TAG_NAME) {
+    return
+}
+
+// Ignore 'go' releases
+if(env.TAG_NAME.startsWith("go/")) {
+    return
+}
```
Comment (pcwiek) [inline Jenkinsfile]: @tperka-catalogicsoftware ? 😉

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+parallelBuildStageList["windows"] = {
+    stage("Windows") {
+        buildForWindows()
+    }
+}
+
+stage('Build') {
+    parallel(parallelBuildStageList)
+}
```
Comment (pcwiek) [inline Jenkinsfile]: I wonder if we can't define it 'inline'? Something like
```
stage('Build') {
  parallel([
    {
      stage('Linux') {
        buildForLinux()
      }
    }, 
    {
      stage('Windows') {
        buildForWindows()
      }
    }
  ])
}
```

**Review by dmalinowski-catalogicsoftware — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (jknyspel-catalogicsoftware) [reply Jenkinsfile]: @tperka-catalogicsoftware said it looks ok to him. I've added some `echo` so that it is clearer why we just `return`.

**Review by pcwiek — APPROVED** [review]

**Comment by pcwiek** [comment]
Wait...

**Review by pcwiek — CHANGES_REQUESTED** [review]
Let's remove GitHub actions from our fork. We only want to have our own CI pipeline execute here

**Review by pcwiek — APPROVED** [review]

**Pull request merged**